assembl/graphql/schema.py

```diff
@@ -662,6 +722,34 @@ def mutate(root, args, context, info):
             for attr, value in kwargs.items():
                 setattr(thematic, attr, value)
             db = thematic.db
+
+            # add uploaded image as an attachment to the idea
+            image = args.get('image')
+            if image is not None:
+                filename = os.path.basename(context.POST[image].filename)
+                mime_type = mimetypes.guess_type(filename)[0]
+                uploaded_file = context.POST[image].file
+                uploaded_file.seek(0)
+                data = uploaded_file.read()
+                document = models.File(
+                    discussion=discussion,
+                    mime_type=mime_type,
+                    title=filename,
+                    data=data)
+                # if there is already an attachment, remove it (but the old
+                # image stays on the server)
+                if thematic.attachments:
+                    thematic.attachments.remove(thematic.attachments[0])
```

Comment:
From the discussions I had with Pauline and Roxana, in the future we want to have a gallery of images associated to a discussion and be able to select an image from this gallery. In this case we don't want to remove the image from the gallery, just remove the association of the image to this thematic. And in this case, the image could potentially be associated to more than one thematic/idea. Right now we don't have an admin interface to remove images indeed.

So what do you want me to do here? Really remove the image from the discussion for now? And when we develop an admin interface to handle a image gallery, put the behavior I currently coded back?